operator/v1/0000_80_csi_snapshot_operator_01_crd.yaml

```diff
@@ -0,0 +1,140 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: csisnapshots.operator.openshift.io
+spec:
+  group: operator.openshift.io
+  names:
+    kind: CSISnapshot
+    plural: csisnapshots
+    singular: csisnapshot
+  preserveUnknownFields: false
+  scope: Cluster
+  subresources:
+    status: {}
+  version: v1
+  versions:
+  - name: v1
+    served: true
+    storage: true
+  validation:
+    openAPIV3Schema:
+      description: KubeAPIServer provides information to configure an operator to
+        manage kube-apiserver.
+      type: object
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: CSISnapshotSpec is the specification of the desired behavior of
+            the CSISnapshot.
```

Comment:
It is desired behaviour of `CSI Snapshot operator` not the snapshot itself.

Reply:
I've updated the description to indicate that this is for the operator and not the snapshot.